src/api.js

```diff
@@ -5,6 +5,10 @@ const host = process.env.VUE_APP_DEFAULT_BACKEND_HOST || 'http://localhost'
 
 const api = {
   getPulls (year) {
+    if (port === 80) {
```

Comment:
port will come as a string from env var, so we need one of:

- `parseInt(port) === 80`, or
- `port === '80'`

Reply:
Would also be nice to add a test for this, if possible.